TOOLS/matroska.py

```diff
@@ -466,8 +464,6 @@ def parse_toplevel(s):
     elif sys.argv[1] == '--generate-definitions':
         generate_C_definitions(sys.stdout)
     else:
-        if sys.version_info.major < 3:
-            raise Exception("Dumping requires Python 3.")
```

Comment:
Suggestion: Move this bit to the top of all python files so they simply error rather than do something strange if run under python 2.

Reply:
I think the shebang itself, as well as python2 being dead, should be enough. If someone goes around doing `python2 *.py` on scripts that are clearly python3 then it should fail on them in strange ways.